homeassistant/components/insteon/schemas.py

```diff
@@ -64,6 +65,14 @@ def set_default_port(schema: dict) -> dict:
     return schema
 
 
+def insteon_address(value: str) -> str:
+    """Validate an Insteon address."""
+    regex = re.compile(r"([A-Fa-f0-9]{2}\.?[A-Fa-f0-9]{2}\.?[A-Fa-f0-9]{2})$")
```

Comment:
Make the regex a constant so you don't have to compile it every time

Reply:
Done.